extensions-contrib/druid-iceberg-extensions/pom.xml

```diff
@@ -298,23 +298,33 @@
       <artifactId>hadoop-aws</artifactId>
       <version>${hadoop.compile.version}</version>
       <scope>runtime</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>com.amazonaws</groupId>
+          <artifactId>aws-java-sdk-bundle</artifactId>
```

Comment:
don't you need any aws dependency? For example in hdfs-storage, where we excluded this, we also added below
```
        <dependency>
          <groupId>com.amazonaws</groupId>
          <artifactId>aws-java-sdk-s3</artifactId>
          <version>${aws.sdk.version}</version>
          <scope>runtime</scope>
        </dependency>
```

Reply:
The metastore only needs the hadoop-aws jar which provides the `org.apache.hadoop.fs.s3a.S3AFileSystem` class to resolve the `s3a` client. The s3 druid extension(which has the aws-java-sdk-s3 dependency) takes care of operations on the objects retrieved by the metastore.